draft-ietf-tls-tls13.md

```diff
@@ -4544,7 +4577,7 @@ bad_record_mac
 record_overflow
 : A TLSCiphertext record was received that had a length more than
   2^14 + 256 bytes, or a record decrypted to a TLSPlaintext record
-  with more than 2^14 bytes.
+  with more than 2^14 bytes (or some other configured limit).
```

Comment:
"negotiated limit" feels more appropriate here

Reply:
It's not negotiated, it's just imposed. The record limit extension is an announcement

Reply:
I read "configured" in this context as something externally configured (as in application/framework configuration), not something configured through Handshake

though I don't agree that it's imposed - if the server doesn't reply with an extension to notify the client that it understood the extension, the limit is not in force. Just the fact that the server needs to answer to extension that client send makes it a "negotiated limit", not "configured limit" or "imposed limit"

Reply:
negotiated still seems better than configured, though neither is great.